src/mem/mem.c

```diff
@@ -130,7 +170,7 @@ void *mem_alloc(size_t size, mem_destroy_h *dh)
 	mem_unlock();
 #endif
 
-	m = malloc(sizeof(*m) + size);
+	m = malloc(mem_header_size + capacity);
```

Comment:
is there another malloc function that gives aligned data ?

Reply:
`malloc` already gives pointer aligned to 8 or 16, depending on the target. There are `memalign` and `posix_memalign`, but they are only necessary if you want alignment higher than that.